designs/feature-toggle.md

```diff
@@ -1,62 +1,57 @@
-Feature Toggle for Unreleased Features
-==============================================
+# Feature Toggle for Unreleased Features
```

Comment:
Thinking of making a `legacy` designs directory. This document is stale but it has historical value. And many of the motivations/concepts still apply.